src/Command/CreateControllerCommand.php

```diff
@@ -66,12 +71,19 @@ private function buildController($name) {
      * @param $output
      */
     private function checkIfFileExists($name, $output) {
```

Comment:
This method is doing too much now:
- check if file exists;
- prints an error message
- exit the program flow

It will be cleaner if you return simple `true` or `false` here and handle the response inside `execute()`. This way you don't even need `$output` here and it is beautiful! 👍 

```suggestion
     private function checkIfFileExists(string $fileName): bool {
```

Notice, I used type hint for `$fileName` and return type for the function.